pyproject.toml

```diff
@@ -2,13 +2,137 @@
 requires = ["setuptools>=42", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "imod"
+version = "0.15.0"
+requires-python = ">=3.10"
+authors = [
+    { name = "Deltares", email = "huite.bootsma@deltares.nl" },
+]
+maintainers = [
+    { name = "Deltares", email = "huite.bootsma@deltares.nl" }
+]
+description = "Make massive MODFLOW models!"
+readme = "README.rst"
+license = { file = "LICENSE" }
+keywords = ["imod", "modflow", "groundwater modeling"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Hydrology",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "opic :: Scientific/Engineering"
+]
+dependencies = [
+    "affine",
+    "cftime >=1",
+    "dask",
+    "fastcore",
+    "Jinja2",
+    "matplotlib",
+    "numba",
+    "numpy",
+    "pandas",
+    "pooch",
+    "scipy",
+    "tomli_w",
+    "toolz",
+    "tqdm",
+    "xarray >=0.15",
+    "xugrid >=0.3.0",
+]
+
+[project.urls]
+Homepage = "https://github.com/Deltares/imod-python"
+Documentation = "https://deltares.github.io/imod-python/"
+Repository = "https://github.com/Deltares/imod-python.git"
+"Bug Tracker" = "https://github.com/Deltares/imod-python/issues"
```

Comment:
Why the quotes around "Bug Tracker"?

Reply:
I followed the example on:
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#advanced-plugins

In that example the "Bug Tracker" is also in quotes